terasoluna-gfw-parent/pom.xml

```diff
@@ -370,6 +370,26 @@
         <type>pom</type>
       </dependency>
       <!-- == End TERASOLUNA == -->
+
+      <!-- == Begin Spring Framework == -->
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>${springframework.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <!-- == End Spring Framework == -->
+
+      <!-- == Begin Spring Security == -->
+      <dependency>
+        <groupId>org.springframework.security</groupId>
+        <artifactId>spring-security-bom</artifactId>
+        <version>${springframework.security.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <!-- == End Spring Security == -->
```

Comment:
- [ ] I think it is better to collect the library of the spring framework.

https://github.com/terasolunaorg/terasoluna-gfw/blob/350055849d4eca13c94a73ed7e346f51c0343874/terasoluna-gfw-parent/pom.xml#L194-L196

Reply:
OK